### PR TITLE
Check

### PR DESCRIPTION
**Pull request by rkwhitlock**

### CODE REVIEW

File: ChessGame/static/chessgame/js/ChessBoard/chessBoard.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ class ChessBoard {
   whiteCaptured = document.createElement('div');
   blackCaptured = document.createElement('div');
```
Comment (rkwhitlock) [inline ChessGame/static/chessgame/js/ChessBoard/chessBoard.js:8]: I am having the hardest time ever checking for opponents being in check and preventing pieces moving in check - I might want to restructure my data due to this

Comment (ntaylorbishop) [reply ChessGame/static/chessgame/js/ChessBoard/chessBoard.js:8]: Oooh this is a really interesting problem. I'd keep another 2D array of bools that holds the state of the board with respect to the King and whether it's in check or not. Definitely not an easy problem to solve.

Comment (ntaylorbishop) [reply ChessGame/static/chessgame/js/ChessBoard/chessBoard.js:8]: You might try looking at open source implementations of Chess as well and see how they do it. Good practice on reading / understanding other people's code

Comment (rkwhitlock) [reply ChessGame/static/chessgame/js/ChessBoard/chessBoard.js:8]: Okay yeah, I looked at some other implementations yesterday but it just looked like I had to refactor the way I structured a lot of my code. The 2D array could work though

Comment (rkwhitlock) [reply ChessGame/static/chessgame/js/ChessBoard/chessBoard.js:8]: Or maybe I'll just store the king's position

**Pull request merged**